pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -359,6 +359,10 @@ func (c *clusterAutoscaler) computeCommand() []string {
 		fmt.Sprintf("--scan-interval=%s", c.config.ScanInterval.Duration),
 	)
 
+	for _, taint := range c.config.IgnoreTaints {
+		command = append(command, fmt.Sprintf("--ignore-taint=%s", taint))
```

Comment:
`--ignore-taint` flag is not present for `cluster-autoscaler@v0.10.2`. You could add a validation that forbids setting the ignoreTaints field for K8s < 1.16 (`cluster-autoscaler@v0.10.2` is used only for K8s < 1.16).

Reply:
I looked into the code again, I am not sure how to refer to the K8s version of the Shoot from that file. No such object having that information seem to be have passed to this method. Any suggestions on how to do it?

Reply:
just a suggestion but botanist should contain most info you need. Here you could get it at `pkg/operation/botanist/clusterautoscaler.go` by passing  `b.Shoot.KubernetesVersion` to `New`.

Reply:
Thank you @kon-angelo I think this should be it.

Reply:
Wait a sec, this should be rather validated in the core validation. We should not allow setting ignoreTags for K8s < 1.16.  You should be able to find example for similar validations in the shoot validation.

Reply:
Oh damn! I was late. I pushed the wrong changes then. Let me revert the changes and check your suggestion.

Reply:
Added the check as per your suggestion @ialidzhikov I could find the references. Thanks!